pyproject.toml

```diff
@@ -24,6 +24,7 @@ extend-select =[
     "C4",
     "T10",
     "FIX",
+    "ISC",
```

Comment:
https://docs.astral.sh/ruff/rules/#flake8-implicit-str-concat-isc

I guess this is ok.